signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/multikey/MultiKeySignerProvider.java

```diff
@@ -144,6 +146,17 @@ public Signer createSigner(final FileBasedSigningMetadataFile metadataFile) {
     }
   }
 
+  @Override
+  public Signer createSigner(final String privateKey) {
+    try {
+      final Credentials credentials = Credentials.create(privateKey);
+      return new CredentialSigner(credentials);
+    } catch (final Exception e) {
+      LOG.error("Unable to construct Raw signer from {}", privateKey);
```

Comment:
I think we should also append `e.getMessage()`.

Reply:
Changed this to better reflect how all other signers are created (i.e. from a MetadataFile, rather than a raw string).